### ISSUE TITLE
Het aantal "Erik's standpunten" klopt niet op de pagina

### ISSUE THREAD

**Issue by reithose** [geensnor/DigitaleTuin#47]
Denk dat dat komt doordat hij het aantal woorden telt ipv de standpunten.

**Comment by Zuijdam** [comment]
Ja daar zat ik dus ook naar te kijken, zie laatste commit

**Comment by reithose** [comment]
Ja, ik zei al dat ik denk dat het komt doordat hij het aantal entries in de woorden lijst telt, ipv in de standpunten lijst.

**Comment by reithose** [comment]
Het zijn er meer dan nul als ik goed tel...

<img width="717" alt="afbeelding" src="https://github.com/geensnor/DigitaleTuin/assets/6649052/e4d65cbe-e883-4219-94c1-f76737475c24">
